lambda/aws-web-crawler-scheduler/lambda.py

```diff
@@ -0,0 +1,148 @@
+import os
```

Comment:
Please add Copyright and license comments at the top of file.

Reply:
Fixed in https://github.com/awslabs/generative-ai-cdk-constructs/pull/474/commits/94869bc149e4ed35dee85ff8a903754189564663